tools-local/tasks/ValidateBinInspectResults.cs

```diff
@@ -52,8 +52,7 @@ public override bool Execute()
             {
                 foreach(var baselineFile in BaselineFiles)
                 {
-                    Regex ignoreRegEx = new Regex(baselineFile.ItemSpec);
-                    IEnumerable<XElement> baselineExcluded = errorRows.Where(f => ignoreRegEx.IsMatch(f.Descendants(s_NameElement).First().Value));
+                    IEnumerable<XElement> baselineExcluded = errorRows.Where(f => Regex.IsMatch(f.Descendants(s_NameElement).First().Value), baselineFile.ItemSpec, RegexOptions.IgnoreCase);
```

Comment:
Based on this StackOverflow [link](https://stackoverflow.com/questions/2439965/case-insensitive-regex-without-using-regexoptions-enumeration), you might be able to keep the old code, except make the first line:

```
Regex ignoreRegEx = new Regex($"(?i){baselineFile.ItemSpec}");
```